src/server.js

```diff
@@ -0,0 +1,10 @@
+const logger = require('~logger');
+const server = require('./index');
+
+const { API_PORT, API_URL } = process.env;
+
+server.listen(API_PORT, () => {
+	logger.info(`App running at ${API_PORT}`);
+});
+
+module.exports = server;
```

Comment:
This makes the file return the same object as `index.js`. Consider the following changes so that this will return the actual http/https server.

```suggestion
const httpServer = server.listen(API_PORT, () => {
	logger.info(`App running at ${API_PORT}`);
});

module.exports = httpServer;
```